### PR TITLE
Fix for focus/scrolling issues on Chrome 61

### PR DESCRIPTION
**Pull request by mathieumg**
Fixes https://github.com/facebook/draft-js/issues/1381

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "test-ci": "NODE_ENV=test npm run lint && npm run flow && npm run test"
   },
   "dependencies": {
-    "fbjs": "^0.8.7",
+    "fbjs": "^0.8.15",
```
Comment (trekare) [inline package.json:33]: LGTM! Maybe worth adding a small polish for line 4: `"version": "0.10.3",`

**Review by trekare — APPROVED** [review]
LGTM! Maybe worth updating the version

File: package.json (hunk 1 above)
Comment (mathieumg) [reply package.json:33]: Weird that 0.10.2 is published and this has 0.10.1. I'll double-check nothing slipped through the cracks. Usually the version is bumped automatically when I publish.

**Comment by trekare** [comment]
LGTM! :)

**Pull request merged**